model/metrics/gen-ai.yaml

```
@@ -0,0 +1,39 @@
+groups:
+  - id: metric_attributes.gen_ai
+    type: attribute_group
+    brief: 'This group describes GenAI metrics attributes'
+    attributes:
+      - ref: error.type
+        requirement_level:
+          conditionally_required: "if the operation ended in an error"
+      - ref: server.address
+        requirement_level: required
+      - ref: server.port
+        requirement_level: required
+      - ref: gen_ai.response.model
+        requirement_level: recommended
+      - ref: gen_ai.request.model
+        requirement_level: recommended
+      - ref: gen_ai.system
+        requirement_level: required
+      - ref: gen_ai.operation.name
+        requirement_level: required
+  - id: metric.gen_ai.tokens.usage
+    type: metric
+    metric_name: gen_ai.tokens.usage
```

nit (pluralization is not usually recommended https://opentelemetry.io/docs/specs/semconv/general/metrics/, and also not used in common metrics):

```suggestion
    metric_name: gen_ai.token.usage
```